src/mobile-token/hooks/use-preemptive-renew-token-mutation.tsx

```diff
@@ -43,5 +52,22 @@ export const usePreemptiveRenewTokenMutation = (userId?: string) => {
         );
       }
     },
+    onError: async (err: any, {token, traceId}) => {
+      notifyBugsnag(err, {
+        errorGroupHash: 'AuthError',
```

Comment:
@reidzeibel Could you please confirm this is the right group hash for mobile-token? :)

Reply:
We have these:

`TokenLoadingTimeoutError`
`GetSignedTokenError`

on this PR : https://github.com/AtB-AS/mittatb-app/pull/4439/files#diff-fd9df4f15f1da5e1656210163837e5dd7992c4790f04cd721596106c8813a7f4

We have no grouping yet for this one, but I think it shouldn't be `AuthError`

Reply:
Fixed